docker/specfile-macos-aarch64.txt

```diff
@@ -0,0 +1,128 @@
+# This file may be used to create an environment using:
```

Comment:
Yeah this was what I got when I ran the  `create-lockfile.sh` on my mac with an M1 chip. It uses all the `aarch` linux versions, and it does seem to work to run on mac.

Maybe it's a bad name for this file. But also, I should probably just drop this file and give instructions on how to build your own docker image on mac with Arm chips, since we're not gonna be using it for any real production stuff.